transactions/mint_tokens.cdc

```diff
@@ -13,14 +13,19 @@ transaction(recipient: Address, amount: UFix64) {
     /// Reference to the Fungible Token Receiver of the recipient
     let tokenReceiver: &{FungibleToken.Receiver}
 
+    /// The total supply of tokens before the burn
+    let supplyBefore: UFix64
+
```

Comment:
We may add the `pre` condition as well to not allow the minting of `0` amount.

Reply:
I don't like having that pre-condition any more, on contract methods and in transactions. it wouldn't make a difference here anyway